src/main/fc/tasks.c

```diff
@@ -230,7 +234,7 @@ static void taskUpdateRxMain(timeUs_t currentTimeUs)
 
     rxStateDurationUs[oldRxState] += executeTimeUs - existingDurationUs;
 
-    schedulerSetNextStateTime(rxStateDurationUs[rxState] / TASK_STATS_MOVING_SUM_COUNT);
+    schedulerSetNextStateTime((rxStateDurationUs[rxState] / TASK_STATS_MOVING_SUM_COUNT) + 5);
```

Comment:
What is 5 doing here, could we have a #define for it?

Reply:
This gets superseded by https://github.com/betaflight/betaflight/pull/11033 but fair point. Fixed.